### DISCUSSION TITLE
Issue with GitHub Copilot Trial and Student Pack

### DISCUSSION THREAD

**Discussion by bakabaka9405** [community/community#112415] [GitHub Education]
### Hi everyone,

Hello,

I recently applied for a one-month free trial of GitHub Copilot. During this period, I also applied for and was approved for the GitHub Student Pack. However, after the one-month free trial of Copilot ended, my access to Copilot was also revoked.

Could you please help me understand why this happened? I was under the impression that the GitHub Student Pack included access to Copilot beyond the trial period.

Thank you for your assistance.

![Snipaste_2024-03-14_15-12-39](https://github.com/community/community/assets/42882043/6afc20d9-c223-45a7-9958-17fb1e1c587d)

![Snipaste_2024-03-14_15-12-50](https://github.com/community/community/assets/42882043/0b0b6789-ecb5-46c0-9a29-5e5fc65fb811)

**Comment by neeleshg23** [comment]
I also am having trouble getting the student pack version of CoPilot after doing the free 1 month trial.

**Comment by RafaelAlvesTech** [comment]
That's how long you stayed in the student pack, I'm a student too and I was scared.

**Reply by neeleshg23** [reply]
Wait, so after paying for Copilot, it is impossible to get the student developer pack benefits?
